<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow { 
background-color: #6291b6 ;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="bas1" native="true">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>140</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCustomplot{
background-color:#6291b6


}</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="bas2" native="true">
    <property name="geometry">
     <rect>
      <x>815</x>
      <y>140</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCustomPlot{
border-style: solid;
  border-width: 4px;
  border-radius: 50px;
  border-color: dark blue;
	padding: 10px;

}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>45</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TELEMETRI</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="temp" native="true">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>640</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="height2" native="true">
    <property name="geometry">
     <rect>
      <x>815</x>
      <y>390</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="lspeed" native="true">
    <property name="geometry">
     <rect>
      <x>815</x>
      <y>640</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>450</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans</family>
      <pointsize>-1</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
	color: #fff;
	font-size: 20px;
	font-weight: bold;
border-style: solid;
 

	padding: 10px;


}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="qml.qrc">
      <normaloff>:/başlatj.png</normaloff>
      <normalon>:/durdurj.png</normalon>:/başlatj.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCustomPlot" name="height1" native="true">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>390</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QQuickWidget" name="quickWidget">
    <property name="geometry">
     <rect>
      <x>1275</x>
      <y>0</y>
      <width>260</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::SizeRootObjectToView</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>455</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans</family>
      <pointsize>-1</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
	color: #fff;
	font-size: 20px;
	font-weight: bold;
	border-style: solid;



	padding: 15px;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="qml.qrc">
      <normaloff>:/filtrelej.png</normaloff>:/filtrelej.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QQuickWidget" name="quickWidget_2">
    <property name="geometry">
     <rect>
      <x>1275</x>
      <y>630</y>
      <width>400</width>
      <height>280</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QquickWidget{
border-style: solid;
  border-width: 4px;
  border-radius: 50px;
  border-color: dark blue;
	padding: 10px;

}</string>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::SizeRootObjectToView</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>535</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
background-color: #052b47;
border-style: solid;
  border-width: 4px;
  border-radius: 50px;
  border-color: #5AED72;
	padding: 10px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="movement">
     <enum>QListView::Snap</enum>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <property name="resizeMode">
     <enum>QListView::Fixed</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2211</width>
      <height>1081</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="qml.qrc">:/background1.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="qml.qrc">:/itutayf0.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>125</red>
          <green>211</green>
          <blue>138</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>125</red>
          <green>211</green>
          <blue>138</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>68</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ITU TAYF 1H&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>10</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3D DURUŞ BİLGİSİ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>568</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GPS HARİTA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>272</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Canlı Yayın</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>70</y>
      <width>78</width>
      <height>50</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>130</y>
      <width>78</width>
      <height>50</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_3">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>190</y>
      <width>78</width>
      <height>50</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>45</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pitch</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>105</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Yaw</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>165</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Roll</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="iot" native="true">
    <property name="geometry">
     <rect>
      <x>1059</x>
      <y>140</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="Irtifa_Farki" native="true">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>640</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="Voltage" native="true">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>390</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>550</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
	color: #fff;
	font-size: 20px;
	font-weight: bold;
border-style: solid;
 

	padding: 10px;


}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="qml.qrc">
      <normaloff>:/ayrılj.png</normaloff>:/ayrılj.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>320</width>
      <height>300</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>600</y>
      <width>571</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
	color: #fff;
	font-size: 20px;
	font-weight: bold;
border-style: solid;
 

	padding: 10px;


}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="qml.qrc">
      <normaloff>:/v.png</normaloff>:/v.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>270</width>
      <height>270</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>38</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>İniş Hızı:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>97</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>27</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HATA KODU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
	color: #fff;
	font-size: 20px;
	font-weight: bold;
border-style: solid;
 

	padding: 10px;


}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="qml.qrc">
      <normaloff>:/op.png</normaloff>:/op.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>805</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
	color: #fff;
	font-size: 20px;
	font-weight: bold;
border-style: solid;
 

	padding: 10px;


}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="qml.qrc">
      <normaloff>:/op.png</normaloff>:/op.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>845</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
	color: #fff;
	font-size: 20px;
	font-weight: bold;
border-style: solid;
 

	padding: 10px;


}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="qml.qrc">
      <normaloff>:/op.png</normaloff>:/op.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
	color: #fff;
	font-size: 20px;
	font-weight: bold;
border-style: solid;
 

	padding: 10px;


}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="qml.qrc">
      <normaloff>:/op.png</normaloff>:/op.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>915</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
	color: #fff;
	font-size: 20px;
	font-weight: bold;
border-style: solid;
 

	padding: 10px;


}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="qml.qrc">
      <normaloff>:/op.png</normaloff>:/op.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>845</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>İrtifa Farkı:</string>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>bas1</zorder>
   <zorder>bas2</zorder>
   <zorder>label</zorder>
   <zorder>temp</zorder>
   <zorder>height2</zorder>
   <zorder>lspeed</zorder>
   <zorder>pushButton</zorder>
   <zorder>height1</zorder>
   <zorder>quickWidget</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>quickWidget_2</zorder>
   <zorder>listWidget</zorder>
   <zorder>logo</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>lcdNumber_2</zorder>
   <zorder>lcdNumber_3</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>iot</zorder>
   <zorder>Irtifa_Farki</zorder>
   <zorder>Voltage</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>label_12</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qml.qrc"/>
 </resources>
 <connections/>
</ui>
